layouts/page/pricing.html

```diff
@@ -547,13 +546,14 @@ <h3>What is an organization? </h3>
             </p>
             <h3>What are projects and stacks? </h3>
             <p>
-                Pulumi <a href="{{ relref . "/docs/intro/concepts/project" }}">projects</a> and <a href="{{ relref . "/docs/intro/concepts/stack" }}">stacks</a> are a way to organize
+                Pulumi <a href="{{ relref . "/docs/intro/concepts/project" }}">projects</a>
```

Comment:
If not tracking these is intentional, we should definitely provide some sort of documentation explaining when and how to use `data-track` attributes. I'll stop commenting on missing attributes for now.